src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/HeaderSWA.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  Button,
+  Column,
+  GridColumns,
+  ResponsiveBox,
+  Spacer,
+  Text,
+  Image,
+} from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import { resized } from "Utils/resized"
+import { useFeatureFlag } from "System/useFeatureFlag"
+import { useAnalyticsContext, useSystemContext } from "System"
+import { useTracking } from "react-tracking"
+
+export const HeaderSWA = () => {
+  const enableSWAInquiryFlow = useFeatureFlag("swa-inquiry-flow")
+  const { user } = useSystemContext()
+  const { contextPageOwnerType } = useAnalyticsContext()
+  const { trackEvent } = useTracking()
+
+  const getInTouchRoute = enableSWAInquiryFlow
+    ? "/sell/inquiry"
+    : "mailto:sell@artsy.net?subject=Inquiry about selling with Artsy"
+
+  const trackStartSellingClick = () => {
+    trackEvent({
+      action: "clickedStartSelling",
+      context_module: "Header",
+      context_page_owner_type: contextPageOwnerType,
+      label: "Start Selling",
+      user_id: user?.id,
+      destination_path: "/sell/submission",
+    })
+  }
+
+  const trackGetInTouchClick = () => {
+    trackEvent({
+      action: "clickedGetInTouch",
+      context_module: "Header",
+      context_page_owner_type: contextPageOwnerType,
+      label: "Get in Touch",
+      user_id: user?.id,
+      user_email: user?.email,
+    })
+  }
+
+  const image = resized(
+    "https://files.artsy.net/images/content-card-swa-landing-page.png",
+    { width: 1104, height: 833 } // TODO: Source image should be 4:3, ideally
```

Comment:
**issue(non-blocker):** is there something we can do about this already? if not we can remove the todo

Reply:
That's not the final image, I'd keep it as a reminder and remove when the design is finalised

Reply:
cool - thanks